data/tribes/initialization/barbarians/units.lua

```diff
@@ -1354,7 +1354,7 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Purpose helptext for a Barbarian production site: Inn
             purpose = pgettext("building", "Prepares rations for scouts and rations and snacks to feed the miners in the basic and deep mines."),
             -- TRANSLATORS: Performance helptext for a Barbarian production site: Inn
-            performance = pgettext("barbarians_building", "The inn can produce one ration in %1$s and one snack in %2$s on average, if the supply is steady."):bformat(format_minutes_seconds(1, 18), format_minutes_seconds(1, 18))
+            performance = pgettext("barbarians_building", "If this building is fully supplied and all kinds of food are needed by the economy, production of one ration takes %1$s and one snack takes %2$s on average. If only one kind of food is needed by the economy, production of one ration takes %3$s and one snack takes %4$s on average."):bformat(format_minutes_seconds(1, 18), format_minutes_seconds(1, 18), format_seconds(37), format_seconds(41))
```

Comment:
```suggestion
            performance = pgettext("barbarians_building", "If this building is fully supplied and all kinds of food are needed by the economy, production of one ration and one snack takes %1$s on average. If only one kind of food is needed by the economy, production of one ration takes %2$s and one snack takes %3$s on average."):bformat(format_minutes_seconds(1, 18), format_seconds(37), format_seconds(41))
```

Reply:
@tothxa your suggestion is incomplete without the usage of the word each

Reply:
I updated the suggestion.

Reply:
this should read one of each food finally I believe

Reply:
It doesn't feel right to me. How about using "one of each of them" here as well? (suggestion edited accordingly)

Reply:
Actually I believe "one of each" was correct.

Reply:
ok